tests/virt_autotest/virsh_external_snapshot.pm

```diff
@@ -66,7 +66,8 @@ sub run_test {
 
         record_info "virsh-snapshot", "Creating Live External Snapshot of guest's memory and disk state";
         #required guests as running status to create Live External Snapshot
-        assert_script_run("virsh start $_") foreach (@vm_hostnames_inactive_array);
+        assert_script_run("virsh start $_ || true") foreach (@vm_hostnames_inactive_array);
```

Comment:
Why "|| true"?

Reply:
Because we wanna have all the guests running but we're not testing it.

Reply:
Why not simply use `script_run` instead?

Reply:
Did so. Thank you

Reply:
I can imagine that you are not testing "virsh start", but your test actually count on it. If it fails, the test shall not go further , or unexpected weird errors may happen. So "assert" actually IS NEEDED, isn't it?

Reply:
I'll do:
```perl
if (script_retry("nmap $guest -PN -p ssh | grep open", delay => 30, retry => 6, die => 0)) {
    record_soft_failure "Reboot on $guest failed";
    script_run "virsh destroy $guest",      90;
    assert_script_run "virsh start $guest", 60;
}
```
@alice-suse is it OK?

Reply:
The change direction is what I recommended :)

Only one concern: vm_hostnames_inactive_array are inactive guests which are supposed to be off. Then with above code, the first nmap will always fail and have the soft_failure recorded. But it is not failure. It is what it should be. Right? Others LGTM.

Reply:
Sorry, I updated my previous suggestion. Now the condition will be true when the guest is online which it should not be.